src/custom-elements.js

```diff
@@ -35,6 +37,7 @@ if (!priorCustomElements ||
 
   /** @type {!CustomElementRegistry} */
   const customElements = new CustomElementRegistry(internals);
+  customElements.enableCustomizedBuiltins = priorCustomElements.enableCustomizedBuiltins;
```

Comment:
If somebody set `enableCustomizedBuiltins` before the polyfill executes, we need to respect that.